### PR TITLE
Improvements: Add base docs link to component element

### PR DESCRIPTION
**Pull request by lciolecki**
When component contains docsUrl in configuration then we create base component link with documentation information.

### CODE REVIEW

File: ui/server/src/main/scala/pl/touk/nussknacker/ui/component/ComponentService.scala [hunk 1]
```diff
@@ -107,6 +107,9 @@ object DefaultComponentService {
   private def getComponentIcon(componentsUiConfig: ComponentsUiConfig, com: ComponentTemplate) =
     componentsUiConfig.get(com.label).flatMap(_.icon).getOrElse(DefaultsComponentIcon.fromComponentType(com.`type`))
 
+  private def getComponentDoc(componentsUiConfig: ComponentsUiConfig, com: ComponentTemplate) =
+    componentsUiConfig.get(com.label).flatMap(_.docsUrl)
```
Comment (mproch) [inline ui/server/src/main/scala/pl/touk/nussknacker/ui/component/ComponentService.scala:111]: let's extract method `componentConfig(config, template)` => it's already used

File: ui/server/src/main/scala/pl/touk/nussknacker/ui/component/ComponentService.scala [hunk 2]
```diff
@@ -199,17 +202,24 @@ class DefaultComponentService private(config: Config,
...
-            .map(_.toComponentLink(componentId, componentName))
+        def createLinks(componentId: ComponentId, component: ComponentTemplate): List[ComponentLink] = {
+          val componentLinks = componentLinksConfig
+            .filter(_.isAvailable(component.`type`))
+            .map(_.toComponentLink(componentId, component.label))
+
+          //If component configuration contains documentation link then we add base link
+          getComponentDoc(uiProcessObjects.componentsConfig, component)
+            .map(ComponentLink.createDocumentationLink)
+            .map(doc => componentLinks ++ List(doc))
```
Comment (mproch) [inline ui/server/src/main/scala/pl/touk/nussknacker/ui/component/ComponentService.scala:213]: should it be last or first?

Comment (lciolecki) [reply ui/server/src/main/scala/pl/touk/nussknacker/ui/component/ComponentService.scala:213]: Hmm, you're right - first will be better.

**Review by mproch — APPROVED** [review]

File: ui/client/cypress/integration/components.ts [hunk 3]
```diff
@@ -111,8 +111,8 @@ describe("Components list", () => {
 
   it("should display usages", () => {
     cy.contains(/^Show used only$/).click()
-    cy.get("[role=row] a").filter((i, e) => /^\d+$/.test(e.innerText)).as("links").should("have.length", 2)
-    cy.get("@links").first().click()
+    cy.get("[role=row] a").filter((i, e) => /^\d+$/.test(e.innerText)).as("links").should("have.length", 3)
+    cy.get("@links")[1].first().click()
```
Comment (JulianWielga) [inline ui/client/cypress/integration/components.ts:115]: ?

File: ui/server/src/main/resources/web/static/assets/components/links/documentation.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 11.79 14"><path d="M0 0h.74v14H0zm1.47 14V0h10.32v14zM3 3.32v1.1h7.37v-1.1zm1.42 2.21v1.1h4.42v-1.1z" fill="#fff"/></svg>
```
Comment (JulianWielga) [inline ui/server/src/main/resources/web/static/assets/components/links/documentation.svg:1]: too big compared to others

Comment (JulianWielga) [inline ui/server/src/main/resources/web/static/assets/components/links/documentation.svg:1]: too big compared to others

Comment (lciolecki) [reply ui/server/src/main/resources/web/static/assets/components/links/documentation.svg]: Changed

**Pull request merged**